### ISSUE TITLE
Tabbing into terminal needs 2 tabs

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#117353]
Testing #116559

Tab from the panel actions into the terminal and notice that you need 2 tab stops to end up in the terminal.

**Comment by isidorn** [comment]
Yeah this was already like that in VS Code stable. So this is terminal behaviour, thus also assigning to @Tyriar 
@Tyriar is there some special reason why we do this? I think the focus on the terminal panel should pass focus inside the terminal?

**Comment by Tyriar** [comment]
The first tab focuses the terminal's buffer and lets the user navigate through it, the second is the input.

**Comment by isidorn** [comment]
@Tyriar ok, I leave this up to you to decide.

I just personaly find it weird that I do not see any visual indication of the focus in the terminal buffer. Or I might be missing something

**Comment by Tyriar** [comment]
A focus color outline could be added around the terminal in this case to be more consistent

**Comment by rzhao271** [comment]
The focus around the buffer was working in Codespaces, but not on Windows. I filed an issue for it here https://github.com/microsoft/vscode/issues/117438.

**Comment by Tyriar** [comment]
Closing as the 2 tab stops are [expected](https://github.com/microsoft/vscode/issues/117353#issuecomment-784343759), the high contrast border is also working fine.
